dc/core/WallWindow.h

```diff
@@ -54,10 +54,8 @@ class WallWindow : public QQuickView
 public:
     /**
      * Create a wall window.
-     * @param sceneRect The portion of the scene to display in pixels.
-     *        The window is also sized accordingly.
      */
-    WallWindow( const QRect& sceneRect );
+    WallWindow( QQmlEngine* engine );
```

Comment:
Very good that you managed to share the engine! I'm not sure why but it caused me troubles when I had tried.
@param docstring and default value = nullptr?